### DISCUSSION TITLE
[Feature request] 现在mac下能用哪个?

### DISCUSSION THREAD

**Discussion by nickeyfff** [vvbbnn00/WARP-Clash-API#117] [Q&A]
好像都不更新了. 没法用现在这个

**Comment by vvbbnn00** [comment]
- clash verge rev: https://github.com/clash-verge-rev/clash-verge-rev (FREE)
- singbox: https://sing-box.sagernet.org/ (FREE)
- shadowrocket: https://apps.apple.com/ke/app/shadowrocket/id932747118 (M1 CPU, 3.99 USD)
- surge: https://www.nssurge.com/ (49.99 USD)
- ...

**Reply by nickeyfff** [reply]
666
